web/src/types/index.ts

```diff
@@ -0,0 +1,44 @@
+export type App = {
```

Comment:
I moved the app type into here. There are a few references that needed to be updated.

Reply:
I like it! Makes it more reusable.